.travis.yml

```diff
@@ -10,6 +10,7 @@ install:
 script:
   - export PATH=$PWD/symlinks:$PATH
   - flake8 .
+  - python setup.py sdist bdist_wheel
   - python -m unittest discover tests
```

Comment:
Can you add it at the end, after the tests?

Reply:
And we also need something like https://github.com/mozilla/libmozdata/blob/master/.travis.yml#L13, to make sure the package is installable via pip.